fe/fe-core/src/main/java/com/starrocks/http/rest/TransactionLoadAction.java

```diff
@@ -57,7 +57,7 @@ public TransactionLoadAction(ActionController controller) {
 
     public static void registerAction(ActionController controller) throws IllegalArgException {
         TransactionLoadAction ac = new TransactionLoadAction(controller);
-        controller.registerHandler(HttpMethod.PUT, "/api/{" + DB_KEY + "}/transaction/{" + TXN_OP_KEY + "}", ac);
+        controller.registerHandler(HttpMethod.POST, "/api/{" + DB_KEY + "}/transaction/{" + TXN_OP_KEY + "}", ac);
```

Comment:
why we put transaction under a database?
why not /api/transaction/begin?

Reply:
Implement in other PR